R/pkg/R/functions.R

```diff
@@ -163,3 +163,17 @@ setMethod("n", signature(x = "Column"),
           function(x) {
             count(x)
           })
+
+#' when
+#'
+#' Evaluates a list of conditions and returns one of multiple possible result expressions.
```

Comment:
The documentation for `when` and `otherwise` look the same to me ? Could we explain the difference a bit better ?